Core/src/main/java/io/htmlcss/model/ReportDataSales.java

```diff
@@ -0,0 +1,41 @@
+package io.htmlcss.model;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.Hashtable;
+
+public class ReportDataSales extends ReportData{
+	private Dictionary<Integer, Float> dPrice = null;
```

Comment:
Doesn't make much sense to extend a class & keep a similar private variable. Maybe make the super class `protected`?